File: JS/1.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    //nums.sort((a,b)=>{return a<b})
    for (let index = 0; index < nums.length; index++) {
        find = target-nums[index]
        f=nums.indexOf(find)
        if(-1!=f && index!=f)
        {
            return [index,f]
        }    
    }
    return [0,0]
};

r = twoSum([3,2,4],6)
console.log(r)

File: Go/1.go
package main

func twoSum(nums []int, target int) []int {
	m:=[]int{}
	s :=len(nums)
	for index:=0;index<s;index++ {
		f := target - nums[index]
		for i,v := range nums{
			if f==v && i!=index {
				m = append(m, index,i)
				return m
			}
		}
	}
	return m
}
//func main(){
//	nums := []int{3,2,4}
//	target :=6
//	twoSum(nums,target)
//}